package.json

```diff
@@ -37,15 +37,15 @@
     "@apollo/react-hooks": "~3.1.5",
     "@native-html/iframe-plugin": "^2.6.1",
     "@native-html/table-plugin": "^5.3.1",
-    "@react-native-async-storage/async-storage": "1.17.11",
-    "@react-native-community/datetimepicker": "6.7.3",
-    "@react-native-community/netinfo": "9.3.7",
-    "@react-native-masked-view/masked-view": "0.2.8",
-    "@react-navigation/bottom-tabs": "5.11.15",
-    "@react-navigation/drawer": "5.12.9",
-    "@react-navigation/native": "5.9.8",
-    "@react-navigation/stack": "5.14.9",
-    "@sentry/react-native": "4.15.2",
+    "@react-native-async-storage/async-storage": "1.18.2",
+    "@react-native-community/datetimepicker": "7.2.0",
```

Comment:
This looks like a major update. Have you read the changelog for this package? How much does it affect our code?

Reply:
It has no effects on the code, it just supports the new possible architecture of react native: https://reactnative.dev/docs/next/the-new-architecture/why